zeroconf/src/macos/browser.rs

```diff
@@ -203,11 +206,14 @@ unsafe fn handle_resolve(
         None
     };
 
+    println!("host_target: {}", c_str::copy_raw(host_target));
```

Comment:
Can you please remove your debug `println!`s

Reply:
Removed